compiler/dyno/lib/uast/post-parse-checks.cpp

```diff
@@ -653,6 +656,27 @@ void Visitor::checkLambdaReturnIntent(const Function* node) {
   }
 }
 
+void
+Visitor::checkProcTypeFormalsAreAnnotated(const FunctionSignature* node) {
+  bool isProcType = (parent() && parent()->isFunction()) ? false : true;
```

Comment:
```suggestion
  bool isProcType = !(parent() && parent()->isFunction());
```
Which also means you can do:
```suggestion
  bool isProcType = !parent() || !parent()->isFunction();
```

Reply:
"Indeed"